tests/protos.rs

```diff
@@ -40,6 +40,10 @@ fn protos() {
     // import our manual builder here so all builder come from the same module in the end user API.
     append_to_file(GRPC_OUTPUT_FILE, "pub use crate::manual_builder::*;");
     append_to_file(GRPC_OUTPUT_FILE, "pub use crate::builder_types::*;");
+    append_to_file(
+        GRPC_OUTPUT_FILE,
+        "pub use crate::qdrant_client::builders::*;",
+    );
```

Comment:
I think we should improve on this eventually. Right now each of them reopens and re-appends the file. But we can do that later.